src/main/scala/it/unibo/authsim/library/dsl/attack/builders/ConcurrentStringCombinator.scala

```diff
@@ -30,7 +32,7 @@ class ConcurrentStringCombinator(val alphabet: List[String], val maxLength: Int)
  * This companion object provides some shorthands for standard alphabets.
  */
 object ConcurrentStringCombinator:
-  def lowercaseLetters: List[String] = List("a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z")
-  def uppercaseLetters: List[String] = lowercaseLetters.map(l => l.toUpperCase)
-  def numbers: List[String] = (0 to 9).map(i => i.toString).toList
-  def symbols: List[String] = List(" ", "!", "\"", "#", "$", "%", "&", "'", "(", ")", "*", "+", ",", "-", ".", "/", ":", ";", "<", "=", ">", "?", "@", "[", "\\", "]", "^", "_", "`", "{", "|", "}", "~")
+  def lowercaseLetters = SymbolicAlphabet(Set("a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"))
```

Comment:
Why don't you use `for i <- 'a' to 'z' yield i ` ? You use all letter of the alphabet here.

Reply:
Well, I didn't think about that and I hadn't remembered to change it. I'll simplify it, thanks!